packages/cypress-mock-openapi/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@heetch/cypress-mock-openapi",
+  "version": "1.0.1",
```

Comment:
Should we bump the version?

Reply:
I did, it was 1.0.0 on the current code.